app/models/financial/Financial.scala

```diff
@@ -22,6 +22,7 @@ import java.time.LocalDateTime
 
```

Comment:
Financial and PaymentFinancial models are the same , so there's a scope for optimisation (Stands out similar to Communication which is used in LatePaymentPenalty and PenaltyPoint)

Reply:
Yeah I see! Good spot, will merge them into one model 😄

Reply:
i'll need to update this on the frontend as well 👍